pkg/controller/migplan/validation.go

```diff
@@ -446,3 +454,17 @@ func (r ReconcileMigPlan) validatePvAction(plan *migapi.MigPlan) error {
 
 	return nil
 }
+
+// Validate Spec.Closed and set status accordingly.
+func (r ReconcileMigPlan) validateClosed(plan *migapi.MigPlan) error {
+	if plan.Spec.Closed {
+		plan.Status.SetCondition(migapi.Condition{
+			Type:     Closed,
+			Status:   True,
+			Category: Critical,
```

Comment:
The `Critical` category is intended to indicate a critical error.  Let's either omit the category here or add a new category.